libcalico-go/lib/apis/v3/ipam_block.go

```diff
@@ -64,6 +64,9 @@ type IPAMBlockSpec struct {
 	// When releasing an IP, passing the sequence number associated with the allocation allows us
 	// to protect against a race condition and ensure the IP hasn't been released and re-allocated
 	// since the release request.
+	//
+	// +kubebuilder:default=0
```

Comment:
Should it be `// +kubebuilder:default:=0`  ?

Reply:
I don't think so? kubebuilder seems to recognize this syntax just fine.. although you're right the docs for kubebuilder do seem to suggest a `:` should be there.

I can add it, but the generator does seem to accept this form :shrug:

Reply:
i'm on the opposite camp -- i never knew the walrus notation could also work other than straight equals :sweat_smile:

Reply:
If it works, then im fine just resolving this :+1: